packages/core/src/payment/payment-request-transformer.ts

```diff
@@ -17,9 +17,12 @@ import { mapToInternalShippingOption } from '../shipping';
 
 import isVaultedInstrument, { isFormattedVaultedInstrument } from './is-vaulted-instrument';
 import Payment, {
+    CreditCardInstrument,
```

Comment:
Same here, do we need to update this bit of code as part of creating this strategy?

Reply:
I'm afraid I don't understand this question. Do you mean the import thing or the whole file?